packages/material/scss/spreadsheet/_layout.scss

```diff
@@ -28,17 +18,6 @@
     // Spreadsheet footer
     .k-spreadsheet-sheets-bar {
         padding: 0;
-
-        .k-button {
-            @extend .k-button-flat;
-            margin: 4px;
-            border-radius: 50%;
-
-            &::before {
-                display: block;
-            }
-        }
-
```

Comment:
The following styles can also be removed, as currently they add additional complexity and are not needed - https://github.com/telerik/kendo-themes/blob/12c7b0534a0a22460408c0037dee162c2185094e/packages/default/scss/spreadsheet/_layout.scss#L239-L243